dashboard/db/migrate/20160510192122_populate_hashed_email_in_users.rb

```diff
@@ -0,0 +1,18 @@
+# A migration that populates (if not already populated) a user's hashed_email
+# for users having an (non-empty) email address.
+
+class PopulateHashedEmailInUsers < ActiveRecord::Migration
+  def change
+    reversible do |direction|
```

Comment:
The guide seems to suggest using `up` and `down` methods are "old style".
http://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-up-down-methods
That said, I'm happy to switch, if you would prefer.